gui/default/assets/lang/lang-en.json

```diff
@@ -151,6 +153,7 @@
    "Help": "Help",
    "Home page": "Home page",
    "However, your current settings indicate you might not want it enabled. We have disabled automatic crash reporting for you.": "However, your current settings indicate you might not want it enabled. We have disabled automatic crash reporting for you.",
+   "If you want to prevent other users on this computer to access Syncthing and through it your files, consider setting up authentication.": "If you want to prevent other users on this computer to access Syncthing and through it your files, consider setting up authentication.",
```

Comment:
"If you want to prevent other users on this computer from accessing Syncthing (and through it, your files)" ...

Reply:
Despite how it looks, that phrase is not part of this PR. Though if it were to be changed, now would be a good time (before it gets too ingrained)

Reply:
https://github.com/syncthing/syncthing/pull/6716